lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -295,6 +298,11 @@ class SPIRVInstTemplateBase : public SPIRVInstruction {
 
   /// Get operand as value.
   /// If the operand is a literal, return it as a uint32 constant.
+  const SPIRVValue *getOpValue(int I) const {
+    return isOperandLiteral(I) ? Module->getLiteralAsConstant(Ops[I])
```

Comment:
Necessity of adding these lines confuses me, I interpret it as you have attempted to translate:
sub_group_non_uniform_broadcast(a, /*constexpr int*/ 42)
and got
OpGroupNonUniformBroadcast ..... /*id*/ 42 /*literal*/

which is quite surprising, as the instruction is declared as _SPIRV_OP(GroupNonUniformBroadcast, true, 6) (aka has 0 literals, unlike for example _SPIRV_OP(GroupNonUniformBallotBitCount, true, 6, false, 1) which has a literal in a position of one).

Reply:
Ah, that's it, thanks!